src/Plugin_Command.php

```diff
@@ -828,7 +831,33 @@ public function get( $args, $assoc_args ) {
 	 */
 	public function uninstall( $args, $assoc_args = array() ) {
 		$successes = $errors = 0;
+
+		$all = Utils\get_flag_value( $assoc_args, 'all', false );
+
 		$plugins = $this->fetcher->get_many( $args );
+
```

Comment:
Hmm, the other `--all` commands do this logic after fetching:
```
		if ( count( $plugins ) < count( $args ) ) {
			$errors = count( $args ) - count( $plugins );
		}
```
which looks like it should be done here too but causes for instance `features/plugin-uninstall.feature:28` to fail with `Error: No plugins uninstalled.` instead of succeeding with `Success: Plugin already uninstalled.`. And also the final step of `features/plugin.feature:3`. So this would be a BC break. But maybe the right thing to do anyway...